web/packages/shared/components/UnifiedResources/shared/PinButton.tsx

```diff
@@ -82,7 +82,6 @@ export function PinButton({
       ) : (
         $content
       )}
-      <HoverTooltip tipContent={tipContent}></HoverTooltip>
```

Comment:
What was going on here? 🤔

Reply:
This was introduced here, and it looks like a simple editing mistake that got overlooked in the review: https://github.com/gravitational/teleport/pull/33635/commits/0457337b38784add0fe8c59bc935546294f21624